lxd/cluster/heartbeat.go

```diff
@@ -281,7 +275,7 @@ func (g *Gateway) heartbeat(ctx context.Context, initialHeartbeat bool) {
 	for _, currentNode := range currentNodes {
 		existing := false
 		for _, node := range allNodes {
-			if node.Address == currentNode.Address && node.ID == currentNode.ID {
```

Comment:
@stgraber should I add the node ID comparison back in here, as its only using the `nodes` table contents now.

Reply:
Yep, looks like the old check is valid again now and should indeed trigger properly with the comparison of both Address and ID